qpsolvers_benchmark/benchmark.py

```diff
@@ -17,10 +18,11 @@
 
 import argparse
 import os
+import sys
 from importlib import import_module  # type: ignore
 
-from qpsolvers_benchmark import Report, Results, TestSet, logging, run
-from qpsolvers_benchmark.plot_metric import plot_metric
+from . import Report, Results, TestSet, logging, run
```

Comment:
Maybe import from specific submodules?

It is a bit more bookkeeping here, but importing from `.`  introduces a constraint for later developers not to import anything from `benchmark.py` in ``__init__.py``.